### PR TITLE
Simplify user reducer

### PR DESCRIPTION
**Pull request by brunomichetti**
- Simplify user's reducer: use for users requests states.
- Change name of authentication reducer.
- Remove unused constants.

### CODE REVIEW

File: src/reducers/user.reducer.js [hunk 1]
```diff
@@ -1,21 +1,16 @@
 import { userConstants } from '../constants/user.constants';
 
-export const authentication = (state = {}, { type, result }) => {
+export const user = (state = {}, { type, result }) => {
   switch (type) {
     case userConstants.USER_REQUEST:
       return {
         userRequest: true,
```
Comment (peter7z) [inline src/reducers/user.reducer.js]: What about `loading`? Or `ongoingUserRequest` if you want to maintain the user request semantics... But I think it sbhould be something that tells me that there is something ongoing/pending.

File: src/components/users/LoginForm.js [hunk 2]
```diff
@@ -22,9 +22,9 @@ const LoginForm = () => {
 
   const [isSubmitted, setIsSubmitted] = useState(false);
 
-  const authState = useSelector((state) => state.authentication);
+  const authState = useSelector((state) => state.user);
```
Comment (peter7z) [inline src/components/users/LoginForm.js]:
```suggestion
  const { userRequest, errorMsg, requestError } = useSelector((state) => state.user);
```

**Review by peter7z — APPROVED** [review]

File: src/components/users/LoginForm.js (hunk 2 above)
Comment (brunomichetti) [reply src/components/users/LoginForm.js]: The point after `userRequest` has to be there?

Comment (peter7z) [reply src/components/users/LoginForm.js]: No, my bad.

**Review by tamarasuarezrod — APPROVED** [review]

**Review by peter7z — APPROVED** [review]
👌

**Pull request merged**